airbyte-integrations/connectors/source-postgres/src/testFixtures/java/io/airbyte/integrations/source/postgres/PostgresTestDatabase.java

```diff
@@ -36,8 +36,27 @@ public static enum BaseImage {
 
   }
 
-  static public PostgresTestDatabase in(BaseImage imageName, String... methods) {
-    final var container = new PostgresContainerFactory().shared(imageName.reference, methods);
+  public static enum ImageModifier {
```

Comment:
This might be pedantic but this should be named ContainerModifier. We're not modifying the image.